### ISSUE TITLE
Firefox HMR error in console

### ISSUE THREAD

**Issue by wesbos** [zeit/next.js#9776]
# Bug report

## Describe the bug

Loading any page in firefox shows this error in the console:

```
The connection to http://localhost:3000/_next/webpack-hmr was interrupted while the page was loading.
```

## To Reproduce

1. `git clone git@github.com:zeit/next.js.git`
1. `cd next.js/examples/blog-starter`
1. `npm install`
1. `npm run dev`
1. Visit localhost:3000 and open console.
1. See `The connection to http://localhost:3000/_next/webpack-hmr?page=/ was interrupted while the page was loading.` 

## Expected behavior

This error should not happen

## Screenshots

<img width="704" alt="Screen Shot 2019-12-18 at 10 43 10 AM" src="https://user-images.githubusercontent.com/176013/71100543-37462e00-2183-11ea-9d2f-079133ba5db7.png">

## System information

- OS: Mac OS Catalina
- Firefox Developer
- Version of Next.js: 9.1.6

## Additional context

Related: https://github.com/zeit/next.js/issues/4456

**Comment by deklanw** [comment]
Just started trying Next.js for the first time and I'm getting this too, I think. Pretty disappointing.

On first load I'm getting:
```
Failed to register/update a ServiceWorker for scope ‘http://localhost:3000/’: Load failed with status 404 for script ‘http://localhost:3000/service-worker.js’.
```

After reload I'm getting this:

```
Firefox can’t establish a connection to the server at http://localhost:3000/_next/webpack-hmr?page=/.
```

and timeouts.

**Comment by timneutkens** [comment]
> Failed to register/update a ServiceWorker for scope ‘http://localhost:3000/’: Load failed with status 404 for script ‘http://localhost:3000/service-worker.js’.

This is because you had a serviceworker installed on localhost:3000 from a project that isn't using Next.js.

**Comment by deklanw** [comment]
@timneutkens Thank you! Removing the old service worker has appeared to fix both issues for me.

**Comment by ijjk** [comment]
Hi, this should be resolved by #9798 which is available in `v9.1.7-canary.1` of Next.js

**Comment by balazsorban44** [comment]
This issue has been automatically locked due to no recent activity. If you are running into a similar issue, please create a new issue with the steps to reproduce. Thank you.
